torchgeo/datasets/south_africa_crop_type.py

```diff
@@ -101,7 +100,7 @@ def __init__(
         paths: str | Iterable[str] = "data",
         crs: CRS | None = None,
         classes: list[int] = list(cmap.keys()),
-        bands: list[str] = all_bands,
+        bands: list[str] = s2_bands,
```

Comment:
I'll let you decide which default value makes the most sense. We can override this in CI if needed in the YAML file.